public/locales/es/common.json

```diff
@@ -159,7 +160,7 @@
     "classic": "Classic v2",
     "cyberpunk": "Cyberpunk",
     "dark": "Mono Dark",
-    "explorer": "Explorer",
+    "explorer": "Explorador",
```

Comment:
I will not rename the theme name, since it is just a proper nouns. I will change it only on japanese or etc

Reply:
Agree on this

Reply:
@turtlemoji done, reverted!